riak_test/src/rtcs.erl

```diff
@@ -326,8 +337,7 @@ lager_config() ->
     {lager,
      [
       {handlers,
-       [
-        {lager_console_backend, debug},
+       [{lager_console_backend, info},
```

Comment:
This config item is unnecessary because the same information is output to console.log and cs will be started by `start` instread of `console`. I want to recommend to remove it.